scripts/deploy-kubefed.sh

```diff
@@ -108,6 +111,15 @@ function kubefed-admission-webhook-ready() {
   [[ "${readyReplicas}" -ge "1" ]]
 }
 
+function deployment-image-as-expected() {
+  local namespace="${1}"
+  local deployment="${2}"
+  local expected_image="${3}"
+
+  local deployed_image="$(kubectl -n ${namespace} get deployment ${deployment} -o jsonpath={.spec.template.spec.containers[0].image})"
```

Comment:
How about using the container name rather than index in case stuff moves around in future?

```suggestion
  local deployment="${2}"
  local container="${3}"
  local expected_image="${4}"

  local deployed_image="$(kubectl -n ${namespace} get deployment ${deployment} -o jsonpath='{.spec.template.spec.containers[?(@.name=="'"${container}"'")].image}')"
```